File: php/fizz-buzz.php
<?php

function fizz_buzz_test()
{
    for($i=1; $i <= 100; $i++){
        if($i % 3 == 0 && $i % 5 == 0){
            echo 'FizzBuzz';
            echo '</br>';
        }
        else if($i % 3 == 0){
            echo 'Fizz';
            echo '</br>';
        }else if($i % 5 == 0){
            echo 'Buzz';
            echo '</br>';
        }else{
            echo $i;
            echo '</br>';            
        }
    }
}

fizz_buzz_test();

?>

File: scala/src/main/scala/Fizzbuzz.scala
object Fizzbuzz extends App {

  def fizzbuzz(n: Int): String = n match {
    case _ if n % 15 == 0 => "FizzBuzz"
    case _ if n % 3 == 0 => "Fizz"
    case _ if n % 5 == 0=> "Buzz"
    case _ => n.toString
  }

  println((1 to 100).map(fizzbuzz(_)).mkString(" "))
}

